webrtc.html

```diff
@@ -9665,8 +9709,8 @@ <h3><dfn>RTCStatsReport</dfn> Object</h3>
     </section>
     <section>
       <h3><dfn>RTCStats</dfn> Dictionary</h3>
-      <p>An <code>RTCStats</code> dictionary represents the stats
-      gathered by inspecting a specific object relevant to a <a>selector</a>.
+      <p>An <code>RTCStats</code> dictionary represents the <a>stats object</a>
+      constructed by inspecting a specific <a>monitored object</a> relevant to a <a>selector</a>.
```

Comment:
"a specific monitored object relevant to a selector"
nit: There are multiple monitored objects relevant to a selector (multiple RTCStats), change to plural.

Reply:
actually, removing the "relevant to a selector". The stats object is constructed based on a single monitored object; the selector is irrelevant to that part of the operation.